app/helpers/Intersect.js

```diff
@@ -300,6 +325,7 @@ async function asyncCheckIntersect() {
     for (const authority of authority_list) {
       try {
         let responseJson = await retrieveUrlAsJson(authority.url);
+        let intersections = await GetStoreData(INTERSECTIONS_DATA, false);
```

Comment:
Add a comment warning that this **might be mutated.** Maybe it even needs a clear name like `updatedIntersections`